instrument/LARMOR_Definition.xml

```diff
@@ -231,6 +231,59 @@
     <algebra val="cyl-approx" />
   </type>
 
+  <type name="WLSF-pixel" is="detector">
+    <cuboid id="shape">
+      <left-front-bottom-point x="-0.001046875" y="-0.0955" z="0.0"  />
+      <left-front-top-point  x="-0.001046875" y="0.0955" z="0.0"  />
+      <left-back-bottom-point  x="-0.001046875" y="-0.0955" z="0.01"  />
+      <right-front-bottom-point  x="0.001046875" y="-0.0955" z="0.0"  />
+    </cuboid>
+    <algebra val="shape" />
+  </type>
+
+ <type name="WLSF-panel">
+   <component type="WLSF-pixel">
+     <locations x="0.03245" x-end="-0.03245" n-elements="32" />
+   </component>
+ </type>
+
+  <component type="DetectorTrolley">
+    <location x="0" y="0" z="25.300" rot="35.0" axis-x="0.0" axis-y="1.0" axis-z="0.0"/> 
+  </component>
+
+  <type name="DetectorTrolley">
+   <component type="LARMORWLSFDetector" idlist="LARMORWLSFDetector">
+     <location z="0.0" /> 
+   </component>   
+  </type>
+ 
+ <type name="LARMORWLSFDetector">
+	<component type="WLSF-panel">
+		<location  x=" 0.2345  " z="  1.090062    " name="WLSF-Panel1"> <facing x="0" y="0" z="25.3"/> </location>
```

Comment:
I don't know if anyone cares about the formatting in these files, but why the extra spaces for the values of `x` and `y`?